src/main/java/ca/yorku/cse2311/tab2pdf/ui/component/EditorTab.java

```diff
@@ -67,9 +67,9 @@
      */
     private final JTextPane EDITOR = new JTextPane();
 
-    private final JTextField TITLE = new JTextField("Not Implemented");
+    public final JTextField TITLE = new JTextField("");
 
-    private final JTextField SUBTITLE = new JTextField("Not Implemented");
+    public final JTextField SUBTITLE = new JTextField("");
```

Comment:
private was the correct modifier to use here. `SUBTITLE` should be a private member, and `getSubtitlefield()` should be used to access it